QA v1.10.0-rc3

EZheln opened the pull request:
Test cleanup:
- standardized tests around export/import handling
- updated tests with Project Monitoring Consumer Group section revamp

Taras-Hlukhovetskyi commented on tests/features/common/actions/api.actions.js:
```diff
@@ -53,8 +53,7 @@ const newJobTemplate = {
 
-const action = {
-  deleteAPIMLProject: async function(
+export const deleteAPIMLProject = async function deleteAPIMLProject (
```
pls fix, save in const anonymus func

EZheln committed:
Test cleanup:
- standardized tests around export/import handling
- updated tests with Project Monitoring Consumer Group section revamp
